core/src/mindustry/world/blocks/defense/turrets/TractorBeamTurret.java

```diff
@@ -24,8 +24,9 @@
     public @Load("block-@size") TextureRegion baseRegion;
     public @Load("@-laser") TextureRegion laser;
     public @Load("@-laser-end") TextureRegion laserEnd;
-
+    
```

Comment:
You accidentally added unnecessary spaces here.

Reply:
meh